test/blackbox/BlackboxTestsLivelinessQos.cpp

```diff
@@ -30,18 +30,17 @@ TEST(LivelinessQos, Liveliness_Automatic_Reliable)
     PubSubWriter<HelloWorldType> writer(TEST_TOPIC_NAME);
 
     // Liveliness lease duration and announcement period
-    uint32_t liveliness_ms = 200;
-    Duration_t liveliness_s(liveliness_ms * 1e-3);
-    Duration_t announcement_period(liveliness_ms * 1e-3 * 0.01);
+    unsigned int lease_duration_ms = 1000;
```

Comment:
I would prefer to use `uint32_t` instead of `unsigned int`. But it is up to you.

Reply:
OK, I'll use `uint32_t` in the future